packages/ag-charts-community/src/module/optionsGraph.ts

```diff
@@ -71,8 +74,17 @@ export class OptionsGraph extends AdjacencyListGraph<unknown, string> implements
         'volumeKey',
     ];
 
+    // If any of these keys are present in the resolved object then calling `clearSafe()` will not clear the graph.
+    private static readonly UNSAFE_CLEAR_KEYS = new Set(['itemStyler']);
```

Comment:
I'm not convinced this is doing anything useful to fix the memory issue, but as above, will address later.